common/src/stack/report-system/command/report/system/tests/test_backend_link.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import paramiko
+import socket
+from stack import api
+
+@pytest.mark.parametrize('backend',
+[host['host'] for host in  api.Call('list host', args=['a:backend'])]
+)
+class TestLinkUp:
+
+	# See if the interfaces stacki configures are actually up on the hosts
+
+		def test_link_status(self, backend):
+			errors = []
+			backend_ssh = paramiko.SSHClient()
+			backend_ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+
+			# Get all interfaces for this host
+			interfaces = [
+				host['interface'] for host in api.Call('list host interface ', args=['a:backend']) if host['host'] == backend
+			]
+
+			try:
+				backend_ssh.connect(hostname=f'{backend}')
+
+			except (paramiko.SSHException, socket.error):
+				pytest.skip(f'Could not ssh into host {backend}')
```

Comment:
Should this be a failure rather than a skip? I'm not 100% sure if the frontend should always be able to ssh into backends, but I would expect that to be a requirement for stacki to work.